### DISCUSSION TITLE
how to limit filltype in multiplayer

### DISCUSSION THREAD

**Discussion by daniloqz** [scfmod/FS22_TerraFarm#96] [Q&A] [answered]
Hello, I have the server with some friends and a few acquaintances, I saw that the fillType is even appearing in some extra crops on the map that shouldn't be possible to dig up from the ground. In light of this, I have no control over preventing someone from selling these products or even knowing if someone sold these products (some of which are very expensive, the value of 1000 liters - example: $7000 for beef). How is this configuration to limit made in the .zip file? I saw that there was a way, but I couldn't identify where to limit it.

**Comment by scfmod** [comment, marked as answer]
In current released versions you unfortunately have to dive into the code and manually adjust/modify, depending on which specific version you are using.
